packages/ai-lab/src/components/AILabObjectDetectionUI.tsx

```diff
@@ -1,7 +1,7 @@
 import * as tf from '@tensorflow/tfjs';
 import React, { useEffect, useRef } from 'react';
 import { ObjectDetectionUIProps } from '../types';
-import { CLASSES } from '../lib/labels';
+import { CLASSES } from './labels';
```

Comment:
How is this supporting custom labels?  It's hardcoded

Reply:
I thought I'd still need to use it as default labels in order to match the data for now. I didn't think the ability to support custom labels was going to be part of this PR.

Reply:
I understand what you're saying.  The task of making labels come in from modelconfig, should mean that the labels passed into modelconfig should be used here, too.   I get breaking it into steps, but I think it's best to fully connect the modelconfig labels here.

Reply:
Requested changes : 
1. Classification model also accepts labels optionally.

https://user-images.githubusercontent.com/53795920/149607203-314e6c72-46fb-4039-94c3-febc6a0569fd.mp4

2. Both models are supporting custom labels in storybook.

https://user-images.githubusercontent.com/53795920/149607247-bc8d93a4-864d-4a14-8f37-fd2a99f23643.mp4